src/test/java/com/jcabi/ssl/maven/plugin/CacertsTest.java

```diff
@@ -56,17 +58,27 @@
      */
     @Test
     public void importsCertificatesFromKeystore() throws Exception {
-        final File keystore = this.temp.newFile("keystore.jks");
-        keystore.delete();
-        final File truststore = this.temp.newFile("cacerts.jks");
-        truststore.delete();
+        final File keystoreFile = this.temp.newFile("keystore.jks");
+        keystoreFile.delete();
+        final File truststoreFile = this.temp.newFile("cacerts.jks");
+        truststoreFile.delete();
         final String pwd = "some-password";
-        new Keystore(pwd).activate(keystore);
-        new Cacerts(truststore).imprt();
+        new Keystore(pwd).activate(keystoreFile);
```

Comment:
composite variable names is a bad practice, see http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html